### PR TITLE
Run mypyc on Windows on tags

### PR DESCRIPTION
**Pull request by ethanhs**
Note this isn't ready for review yet, I want to test running mypyc actually works.

### CODE REVIEW

**Comment by ethanhs** [comment]
Ugh, the run is just erroring with "Command executed with exception: ". Guess I will need to figure out how to debug this...

@msullivan if you have any hints/suggestions they are more than welcome :P

File: appveyor.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ install:
 build: off
 
 test_script:
+    - ps: "If ($env:APPVEYOR_REPO_TAG -Match \"false\") { Set-Alias -Name python3 -Value \"$env:PYTHON\\python.exe\"; python3 -m pip install -r mypy-requirements.txt; $env:CC=\"clang\"; $env:MYPYC_OPT_LEVEL=\"0\"; python3 setup.py --use-mypyc build_ext --inplace }"
```
Comment (ethanhs) [inline appveyor.yml]: Note to self: change this to `-Match "true"` when done testing.

**Comment by msullivan** [comment]
What if you change the optimization level to "d" from "0"? That seems to be what it is complaining about the most.

**Comment by msullivan** [comment]
we don't use appveyor anymore so I'm gonna close this one

**Pull request closed without merging**